prisma/schema.prisma

```diff
@@ -8,13 +8,20 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+
+enum keyword_status {
+  Inprogress @map("in-progress")
+  Done @map("done")
```

Comment:
There could be an additional status:

```suggestion
  Done @map("done")
  Failed @map("failed")
```

Web scraping is error-prone. Say, Google rate limiting changes, it could result in lots of failures and such failures must be tracked at the keyword level.